addons/account/models/res_config_settings.py

```diff
@@ -87,6 +87,8 @@ class ResConfigSettings(models.TransientModel):
     use_invoice_terms = fields.Boolean(
         string='Default Terms & Conditions',
         config_parameter='account.use_invoice_terms')
+    # Techinal field to hide country specific fields from configuration
```

Comment:
```suggestion
    # Technical field to hide country specific fields from configuration
```

Reply:
@mpa-odoo Please recheck!!!  Can you rebase, check if there are maybe other fields we forgot, ...   I think we can merge this on Monday then.  @rgo-odoo

Reply:
@jco-odoo I think it should be ok now, can you check ? Thanks.

Reply:
see review